honeybadgermpc/betterpairing.py

```diff
@@ -717,3 +717,45 @@ def zero():
     @staticmethod
     def one():
         return ZR(PyFr("1"))
+
```

Comment:
I am not okay with this change. We have a whole module dedicated to polynomial operations - `polynomial.py`. We should try to make use of that instead of duplicating functionality just because we are operating on a different type.

Reply:
That module actually caches interpolations and here you don't get that benefit. I would strongly suggest using that code since that contains similar operations in one file making the code more maintainable in the future.

Reply:
@tyurek

Reply:
@tyurek are these methods still needed?

Reply:
I thought these were moved these to the polynomial class?